### ISSUE TITLE
JWT(Access token as JWT) returning empty scope

### ISSUE THREAD

**Issue by ldeveloperl1985** [GluuFederation/oxAuth#956]
We are getting empty scope after decoding JWT. Below is oxd `get-client-token` API request.
```
{
    "scope": [
        "openid",
        "oxd"
    ],
    "access_token": "eyJraWQiOiIzNjgwYTY4OC0xYmE4LTRjM2YtOTgwNy1hNmZkZDg0MDllOWIiLCJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJodHRwczovL2dsdXUubG9jYWwub3JnIiwiYXVkIjoiQCFBOTdBLjZGOTAuQzFGRS45QUMyITAwMDEhRTQxOC44RTk1ITAwMDghQTdEMC42RUJGLkNCMDEuRjhFNSIsInNjb3BlIjpbXSwiY2xpZW50X2lkIjoiQCFBOTdBLjZGOTAuQzFGRS45QUMyITAwMDEhRTQxOC44RTk1ITAwMDghQTdEMC42RUJGLkNCMDEuRjhFNSIsInRva2VuX3R5cGUiOiJiZWFyZXIiLCJleHAiOjE1NDM4NDUxMTEsImlhdCI6MTU0Mzg0NDgxMSwic3ViIjoiSGxEaEtXc2lKUG1rQXdseFhvdWV1QWdzMGVrVVhWYWRfUUYxdE0wWE45SSJ9.TDL1DQsDm7MysKGScVNNUnnXQobu6vBPxV_l4J1l2RRHoFxaRdv78vjXk92ox9njxJtKDOH5xwDQS5Ygyd9mWVxZmz2OvMNLvidTSdMJkHtaY1Eyj_E40G81sQl_u5NI5r3AptKkcedM9l7ybE9kbCYsUOsAGqdHnm5BVkoCFk-R_I-7RqmPAQZnz7nIjVohW3XnP_N87tk_kh8LqFZ_SA4mef_AdgmA9mZf1j6f_pDvJ6uknd9df8geR1s-wq_eu9ft0NyiHBw2xzC9vJoab1gDn32Y5uH0u_ahzS4LZhkCfMwxWciNhzXSuQXHBgXXWdf6InpBrTWtKK0Kv0Zj1A",
    "expires_in": 299,
    "refresh_token": null
}
```

After decode (using `https://jwt.io/`)
```
{
  "iss": "https://gluu.local.org",
  "aud": "@!A97A.6F90.C1FE.9AC2!0001!E418.8E95!0008!A7D0.6EBF.CB01.F8E5",
  "scope": [],
  "client_id": "@!A97A.6F90.C1FE.9AC2!0001!E418.8E95!0008!A7D0.6EBF.CB01.F8E5",
  "token_type": "bearer",
  "exp": 1543845111,
  "iat": 1543844811,
  "sub": "HlDhKWsiJPmkAwlxXoueuAgs0ekUXVad_QF1tM0XN9I"
}
```

-------------------------------------------------------------

scope is empty so is it oxd issue or oxauth issue?

**Comment by yuriyz** [comment]
I've checked it on latest oxauth 3.1.5. Scopes are inside correct, see screenshot.

![image](https://user-images.githubusercontent.com/2013070/49379751-8e17cb80-f718-11e8-9d8d-cd750293c754.png)

**Comment by ldeveloperl1985** [comment]
I've tested it on ce-dev3 3.1.5, but got the same problem.
[oxd-server.log](https://github.com/GluuFederation/oxAuth/files/2639910/oxd-server.log)

**Comment by yuriyz** [comment]
I was able to reproduce it. It was reproducible for client authentication only. Fixed in 3.1.5.
